rules/parser.y

```diff
@@ -235,6 +237,28 @@ extra_labels_opts  :
                      { $$ = true }
                    ;
 
+vector_matching    :
+                     { $$ = nil }
+                   | GROUP_OP '(' label_list ')'
+                     { 
+                       var err error
+                       $$, err = newVectorMatching("", $3, nil)
+                       if err != nil { yylex.Error(err.Error()); return 1 }
+                     }
+                   | GROUP_MOD GROUP_OP '(' label_list ')'
```

Comment:
Unfortunately this still is another runtime error as one can still use dumb labels in GROUP_MOD.